hyundai_canfd.dbc

```diff
@@ -512,6 +530,8 @@ BO_ 1041 DOORS_SEATBELTS: 8 XXX
  SG_ PASSENGER_SEATBELT : 36|1@0+ (1,0) [0|1] "" XXX
 
 BO_ 1043 BLINKERS: 8 XXX
+ SG_ LEFT_LAMP_ALT : 8|1@0+ (1,0) [0|1] "" XXX
+ SG_ RIGHT_LAMP_ALT : 10|1@0+ (1,0) [0|1] "" XXX
```

Comment:
ah this is why I didn't see any blinkers for haram's routes! is there any bit that's common across all cars?

Reply:
you can use get_url and paste the url in PJ

Reply:
you can use get_url and paste the url in PJ. @deanlee can we make it easy to load arbitrary URLs files in cabana? creating the correct folder structure is tedious